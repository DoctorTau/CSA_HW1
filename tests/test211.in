131
4828 7563 5738 -5102 -1284 7101 265 4090 7892 2979 -1095 2964 -8713 1660 1685 9718 -9721 -4525 -7850 -3373 -9162 6888 1244 3615 5172 -7574 5809 -2065 7671 8297 -6399 1344 9711 682 9485 -7487 -665 4886 7648 6273 6295 9708 268 7490 -6875 -8892 9266 -2773 9614 7570 -180 8109 -1599 -7506 7806 9948 -4170 2416 -2378 -5069 -6036 7664 -2289 -340 2030 -9287 981 2832 -8875 2909 4888 -7094 2993 -8613 -4276 -290 -9028 5595 8950 -1927 3514 -3616 -2325 -6898 6934 -1622 -3724 3752 -4957 -9464 2670 2642 8697 -4349 3081 -4482 9327 -22 -6906 9470 6606 466 6737 1866 -8879 3478 -5282 -8785 -967 -4029 -2579 912 -2041 -75 -8987 -5833 8065 -8250 1700 -1672 -3458 -8938 4849 -6635 9463 -1041 1118 5242 -6615 1869 -8112 