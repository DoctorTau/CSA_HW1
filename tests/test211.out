-5102 -1284 -7850 -9162 -7574 -8892 -180 -7506 -4170 -2378 -6036 -340 -7094 -4276 -290 -9028 -3616 -6898 -1622 -3724 -9464 -4482 -22 -6906 -5282 -8250 -1672 -3458 -8938 -8112 